infrastructure/sysdig-team.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: ops.gov.bc.ca/v1alpha1
+kind: SysdigTeam
+metadata:
+  name: 9c33a9-sysdigteam
+  namespace: 9c33a9-tools
+spec:
+  team:
+    description: The Sysdig Team for the OpenShift Project Set 9c33a9 (PRIME)
+    users:
+      - name: roya.shourouni@nttdata.com
+        role: ROLE_TEAM_EDIT
+      - name: itchison@gmail.com
```

Comment:
Just wanted to confirm that Jason prefers his Gmail account rather than NTT DATA account?